### ISSUE TITLE
Increased training losses

### ISSUE THREAD

**Issue by Alexander8848** [eric-yyjau/pytorch-superpoint#129]
![Image](https://github.com/user-attachments/assets/abd7967e-907f-4960-a458-b0f537c4cb0e)Hi author, I am doing the process of de-exporting coco pseudo-labels with magicpoint, and then using the coco dataset as well as the pseudo-labels to continue training magicpoint under the magicpoint pre-trained model on the synthetic dataset, and the loss of training keeps going up, why is that?
